charts/cluster-api-runtime-extensions-nutanix/templates/cluster-autoscaler/manifests/helm-addon-installation.yaml

```diff
@@ -9,7 +9,7 @@ metadata:
 data:
   values.yaml: |-
     ---
-    fullnameOverride: "cluster-autoscaler-{{ `{{ .Cluster.Name }}` }}"
+    fullnameOverride: "cluster-autoscaler-{{ `{{ md5sum (printf "%s/%s" .Cluster.Namespace .Cluster.Name) }}` }}"
```

Comment:
please leave a comment about whats going on here because its confusing and can help developers down the line.

Reply:
This is no longer relevant.